addons/website_event/models/event_event.py

```diff
@@ -527,10 +527,18 @@ def _search_get_detail(self, website, order, options):
             mapping['description'] = {'name': 'subtitle', 'type': 'text', 'match': True}
         if with_date:
             mapping['detail'] = {'name': 'range', 'type': 'html'}
+
```

Comment:
can we also add a test that fail if we do this ?
```py
return [('address_search', 'ilike', search_term)]
```

(so the user can't read the partner, but he can read the event)

Reply:
Hello @std-odoo 

Can you explain this point in detail? What is the purpose of this test case in the context of search_in_address? 

Thank!

Reply:
@maan-odoo to test that we bypass the access rules on res.partner but not on event.event

Reply:
> can we also add a test that fail if we do this ?
> 
> ```python
> return [('address_search', 'ilike', search_term)]
> ```
> 
> (so the user can't read the partner, but he can read the event)

Hi @std-odoo,
As you can see in the new [commit](https://github.com/odoo/odoo/pull/89796/commits/acd4176da4bb86eb6e69736298a171717c681737#diff-d8271bbe6114dd3bb8305ff3ec64f68e987ff19dbf8bbc827d00257d8488f88eR141-R147), I have used the method '_search_get_detail' to test the 'search_in_address' function implicitly. I don't think it is the correct approach concerning the test case Could you please have a look at it? Could you guide me to the correct approach?

Thanks